docs/how-to/configure-revision-history.md

```diff
@@ -0,0 +1,25 @@
+# How to configure revision history
+
+You can limit how many versions of the images are kept in OpenStack glance by specifying the
```

Comment:
Is there a default value for this? Should the user be warned about it in this how-to?